docs/Installation.md

```diff
@@ -41,9 +41,14 @@ Once homebrew is installed, you can install these missing utilities by running t
 $ brew install coreutils squashfs
 ```
 
+For Debian-based Linux systems, you can install these utilities by running:
+```
+sudo apt-get install coreutils squashfs-tools
+```
+
 ## Nerves Bootstrap
 
-With Elixir, Erlang, and your host utilities installed, you can now add the `nerves_bootstrap` archive to your mix environment. This archive allows Nerves to bootstrap the Mix environment, ensuring that your code is properly compiled using the right cross-compiler for the target. The `nerves_bootstrap` archive also includes a new project generator, which you can use to create new Nerves projects. To install the `nerves_bootstrap` archive:
+With Elixir, Erlang, and your host utilities installed, you can now add the `nerves_bootstrap` archive to your Mix environment. This archive allows Nerves to bootstrap the Mix environment (i.e. fetch the System and Toolchain for your target), ensuring that your code is properly compiled using the right cross-compiler for the target. The `nerves_bootstrap` archive also includes a new project generator, which you can use to create new Nerves projects. To install the `nerves_bootstrap` archive:
```

Comment:
> (i.e. fetch the System and Toolchain for your target)
Fetch actually happens as a side effect of compiling a system or toolchain. Bootstrap ensures that the System and Toolchain are compiled first before any other code,